pkg/utils/validator/clusterresourceplacementeviction.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2025 The KubeFleet Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package validator provides utils to validate ClusterResourcePlacementEviction resources.
+package validator
+
+import (
+	"fmt"
+
+	fleetv1beta1 "github.com/kubefleet-dev/kubefleet/apis/placement/v1beta1"
+	"k8s.io/apimachinery/pkg/util/errors"
+)
+
+// ValidateClusterResourcePlacementEviction validates cluster resource placement eviction fields and returns error.
+func ValidateClusterResourcePlacementEviction(crpe fleetv1beta1.ClusterResourcePlacementEviction) error {
```

Comment:
The logic in this function is already by CEL validation on object creation https://github.com/kubefleet-dev/kubefleet/blob/main/apis/placement/v1beta1/eviction_types.go#L76-L88